api/ask_astro/rest/controllers/get_request.py

```diff
@@ -15,15 +16,23 @@
 
 
 @openapi.definition(response=AskAstroRequest.schema_json())
-async def on_get_request(_: Request, request_id: UUID):
+async def on_get_request(request: Request, request_id: UUID) -> json:
     """
     Handles GET requests to the /requests/{request_id} endpoint.
+
+    :param request: The Sanic request object.
+    :param request_id: The unique identifier for the AskAstro request.
     """
-    logger.info("Received GET request for request %s", request_id)
-    request = await firestore_client.collection(FirestoreCollections.requests).document(str(request_id)).get()
-    logger.info("Request %s exists: %s", request_id, request.exists)
+    try:
+        logger.info("Received GET request for request %s", request_id)
```

Comment:
Should we consolidate how we format string in logger?

Reply:
changed it: https://github.com/astronomer/ask-astro/pull/61/commits/b046f9131f44c5fd7026927fd2abe53a6379cda8